jbi/jira/service.py

```diff
@@ -257,7 +259,7 @@ def add_jira_comment(self, context: ActionContext):
 
         issue_key = context.jira.issue
         formatted_comment = (
-            f"*({commenter})* commented: \n{{quote}}{comment.body}{{quote}}"
+            f"*{commenter}* commented: \n{markdown_to_jira(comment.body or "")}"
```

Comment:
Not something to address in this PR, but it's slightly annoying to have to account for a `comment.body` being `None`. I wonder if that's something that's even possible from the Bugzilla side. Like I'd almost consider a comment without a body a malformed payload and something we should refuse (but still return a `200` for webhook `ack` purposes)

Reply:
Yes
And I think this relates to https://github.com/mozilla/jira-bugzilla-integration/issues/234